opm/simulators/utils/SerializationPackers.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+  Copyright 2019 Equinor AS.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef SERIALIZATION_PACKERS_HPP
+#define SERIALIZATION_PACKERS_HPP
+
+#include <opm/common/utility/MemPacker.hpp>
+
+namespace boost { namespace gregorian { class date; } }
```

Comment:
That's a little bit presumptuous since we don't control Boost.DateTime, but I suppose the interface is stable enough by now to not have to worry too much about forward declaring the type.

Reply:
we have been forwarding in SimulatorTimer.hpp for years so yeah, i don't think this is a real concern. but i'll wait for your opinion on the above before concluding.